packages/composite-checkout/src/types.ts

```diff
@@ -124,7 +124,7 @@ export interface CheckoutProviderProps {
 	children: React.ReactNode;
 }
 
-export type CheckoutType = 'jetpack' | 'akismet' | undefined;
+export type SitelessCheckoutType = 'jetpack' | 'akismet' | undefined;
```

Comment:
💯 This is a great type, thanks! However, the `@automattic/composite-checkout` package is a general purpose UI framework (see its [README](https://github.com/Automattic/wp-calypso/blob/136443bd1dfb5792563475a4a4981e0ffded1750/packages/composite-checkout/README.md#L17) for more details) and doesn't know anything about WPCom, Jetpack, etc. It just creates a step UI and provides callbacks for payment methods. 

I think `SitelessCheckoutType` might be a good type to put into either calypso itself (the `client` directory somewhere) or the `@automattic/wpcom-checkout` package which is a tool chest for use by calypso checkout.

Reply:
Thank you for the extra context, i'll move this to the `wpcom-checkout` package